activity_calendar/models.py

```diff
@@ -18,6 +18,8 @@
 # Models related to the Calendar-functionality of the application.
```

Comment:
(Due to lack of a better place to put this)
The admin panel for ActivityMoment is not really user friendly, so it would be nice to include that in this PR as it's somewhat related and only a few lines.
There's also no __str__ method for ActivityMoment which is part of this problem.